src/components/graph-view.js

```diff
@@ -1291,14 +1291,39 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
     return true;
   };
 
+  shouldGraphZoom = () => {
+    const svgNode = d3
+      .select(this.viewWrapper.current)
+      .select('svg')
+      .node();
+
+    try {
+      const doesHeightValueExist =
+        svgNode.height &&
+        svgNode.height.baseVal &&
+        svgNode.height.baseVal.value;
+      const doesWidthValueExist =
+        svgNode.width && svgNode.width.baseVal && svgNode.width.baseVal.value;
+
+      return (doesHeightValueExist && doesWidthValueExist) || true;
+    } catch (err) {
+      // fail gracefully and still display hidden entity elements even though the first zoom hasn't happened
+      if (this.entities && this.entities.style.visibility === 'hidden') {
```

Comment:
I need to do more testing around these checks in general, but specifically want to ensure that this actually does what we want it to do. Just wanted to get it up to get feedback

Reply:
Yup I think that will work, much nicer looking solution. I'll update!